src/lib/api.ts

```diff
@@ -41,19 +53,22 @@ export const makeFilterUrl = (path: string, query: URLSearchParams) => {
 	};
 };
 
-export const loadDataset: Load = async ({ params }) => {
+export const loadDataset: Load = async ({ params, fetch }) => {
 	const { datasetId } = params;
 	const dataset = await get(`package_show?id=${datasetId}`);
 	const showcases = await get(`ckanext_package_showcase_list?package_id=${datasetId}`);
+	const res = await fetch(`/dataset/${datasetId}.jsonld?profile=${schemaOrgProfile}`);
```

Comment:
This seems wrong - it looks like you are calling the path `/dataset/${datasetId}.jsonld?profile=${schemaOrgProfile}`, without the ckan url, or am I missing something?